react-client/dist/css/custom.css

```diff
@@ -1,12 +1,49 @@
+/*body{
+	background-image: url('../photos/pexels-photo-236401.jpeg');
+}*/
+.navbtn{
+	margin-top: 20px;
+	width: 50%;
+	background-color: lightblue;
 
+}
+.img{
+	width: 95%;
+	
```

Comment:
You'll want to clean up this file at some point by removing unnecessary blank lines like these.